README.md

```diff
@@ -256,6 +258,51 @@ Personal API keys only need read permission to Query. When creating a personal A
 
 Once these are set (and you've restarted the app), you should see events flowing into the PostHog dashboard.
 
+### Connecting with MITxOnline
+
+Set up the [mitxonline](https://github.com/mitodl/mitxonline) as indicated there. _Note: A working OpenEdx installation is not necessary for integration with Learn._
```

Comment:
The MITx Online README should probably indicate that keycloak and apisix are required for the login button to work correctly, otherwise the `/admin` page can be used to log in.  Maybe mention that MITxOnline shouldn't be started up with apisix/keycloak here (instead of or in addition to doing so in the tip below) and  that logging in via /admin should be done if anything needs to be set up like CMS pages.

Reply:
FWIW, even after setting up the env values below, I get into an endless loop when clicking the MITx Online login button.  Ditto for using MITx Online's own keycloak/apisix instance.

Reply:
> Not sure if this PR should also enable mitxonline users to log in via mit-learn's apisix/keycloak instance, but if so I could not validate that because something seems off with my local setup and I was unable to log in that way.

Do you mean click login via MITxOnline login button?  I had forgotten that does a surprising infinite loop. I'm not sure what causes that, but logging in via MIT Learn works.

@jkachel Do you have any idea? IMO not critical though.